.travis.yml

```diff
@@ -13,6 +13,8 @@ env:
     - SYMFONY__PHPCR__TRANSPORT=doctrinedbal
     - TEST_FLAGS=""
     - SYMFONY__DATABASE__DRIVER=pdo_mysql
+    - SYMFONY__DATABASE__CHARSET=utf8mb4
+    - SYMFONY__DATABASE__COLLATE=utf8mb4_unicode_ci
```

Comment:
added that travis is correctly failing when somebody increase the field: example: https://travis-ci.org/sulu/sulu/jobs/335311387